ceph.spec.in

```diff
@@ -3,6 +3,7 @@
 %bcond_with tests
 %bcond_without tcmalloc
 %bcond_without libs_compat
+%bcond_with librocksdb
```

Comment:
ok, thanks for explaining.

@liewegas , you added this line in 213e0acf7e81aad1d7d24b0577463b80fc8a095b, is it ok to conditionalize this out now (as this PR does)?